config/jobs/GoogleCloudPlatform/k8s-cloud-provider/k8s-cloud-provider.yaml

```diff
@@ -0,0 +1,20 @@
+presubmits:
+  GoogleCloudPlatform/k8s-cloud-provider:
```

Comment:
is there a reason we're using prow.k8s.io?
will it be under kubernetes-sigs later?

otherwise this should probably be on the general google OSS prow, googlers can check go/which-prow
https://github.com/GoogleCloudPlatform/oss-test-infra

Reply:
This instance of test-infra seemed more appropriate since there are other GoogleCloudPlatform projects specified here. 

Looking at go/which-prow, I would say this falls under prow.k8s.io since it is a "Kubernetes-adjacent" project. Specificaly, it is vendored by another repo under kubernetes/

Reply:
There are many projects here by legacy that should not have been.

IIRC the document actually says "Kubernetes adjacent _CNCF project_"

prow.k8s.io being under google.com is a legacy bug, it will be moving to wg-k8s-infra finding and management.

Reply:
Ah ok, missed the CNCF part. Will close this and open one in the other repo. Thanks!